### DISCUSSION TITLE
HP 830 G6 Extended display

### DISCUSSION THREAD

**Discussion by Dan1912** [community/community#101411] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

hello i am currently running mac os ventura on
hp 830 g6 i5-8365u  intel uhd 620 whiskeylake
everything works perfectly including hdmi out with exernal dispay mirror option enabled
the problem occurs when i enable extended display option, the internal  main display lags alot after running programs...
any help will be apreciated thanks....
![Screenshot 2024-01-22 at 15 16 00](https://github.com/community/community/assets/156875696/3b15e273-4d33-4582-b7df-b9c0aac9fc71)

**Comment by Chupriarti** [comment]
Seems like nothing can be done here. Have faced some similar reports about this problem. You can hope this will be fixed with future updates from Apple.
